src/common.speech/DialogServiceAdapter.ts

```diff
@@ -501,7 +501,7 @@ export class DialogServiceAdapter extends ServiceRecognizerBase {
                 MessageType.Binary,
                 "audio",
                 this.privDialogRequestSession.requestId,
-                null,
+                "audio/x-wav",
```

Comment:
Wondering about removing this change.. could this cause any issues for future formats? I've just tested and it works with 'null'

Reply:
It depends on the endpoint and how strict it is when it transforms the audio.

The native code sends the header.

The JS code got burned when some endpoints' transformation service started demanding the header.

If we stop transmitting PCM, we're going to have a lot of work to do anyways around the compression.

---
In reply to: [399573404](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/165#discussion_r399573404) [](ancestors = 399573404)